### PR TITLE
did all the required changes in the login page

### PR DESCRIPTION
**Pull request by Sloth-Panda**
## Issue that this pull request solves

 Closes: #226 

## Proposed changes
 Replace current image on login page with the required image.
 Make the background of the page white
 Make the Sign In button same as submit button on contact-us page
 Add hover effect to the icons of the input box with proper hovering effect

### Brief description of what is fixed or changed
 Replaced current image on login page with the required image
 Made the background of the page white
 Made the Sign In button same as submit button on contact-us page
 Added  hover effect to the icons of the input box with proper hovering effect

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot (83)
This is how it looks after making all the four proposed changes you asked for

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Sloth-Panda please remove the background color from the left side picture and also the default color of the eye icon should be black(same as other icons)

**Comment by Sloth-Panda** [comment]
@Kajol-Kumari  the eye icons color is like other icons color only I have taken the snap when I hovered over the eye icon to represnt thr changes on hovering

Everything else is fine? I will change the background color of the left side

**Comment by Sloth-Panda** [comment]
Screenshot (86)

You are asking for this right? @Kajol-Kumari

**Comment by Sloth-Panda** [comment]
Has made it like this you can check

File: frontend/src/components/auth/login/login.css [hunk 1]
```diff
@@ -65,11 +67,12 @@
   box-shadow: inset 2px 2px 5px #888888, inset -2px -2px 5px #ffffff;
 }
 
-.inputLogin:focus{
-  box-shadow: inset 2px 2px 5px #1E488F, inset -2px -2px 5px #1E488F;
+.inputLogin:focus {
```
Comment (Kajol-Kumari) [inline frontend/src/components/auth/login/login.css:70]: Just make this color same as the linear gradient we are using for input fields click on other pages

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Please make this minor change. Rest LGTM :+1:

File: frontend/src/components/auth/login/login.css (hunk 1 above)
Comment (Sloth-Panda) [reply frontend/src/components/auth/login/login.css:70]: @Kajol-Kumari  which exact one's color?

**Comment by Sloth-Panda** [comment]
has made the input text-fields color same as other pages text-fields color

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1: 

Can you please let me know if you are DWOC or SWOC participant?

**Comment by Sloth-Panda** [comment]
Hey, @Kajol-Kumari  actually I am a participant of both so if you can add tags of both DWOC and SWOC, that will be help

**Pull request merged**